createPR.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+
+set -eu
+
+readonly SIMONE_ROOT="/var/www/Simone"
+readonly AUTHOR_EMAIL="yunobot@some.domain.tld"
+
+readonly REPO="alexAubin/doc"
+readonly REPO_KEY="$SIMONE_ROOT/config/id_rsa"
+readonly REPO_TOKEN=$(cat $SIMONE_ROOT/config/github_token)
```

Comment:
shouldn't all of those be in a configuration file or something to avoid having to stash when pulling?